homeassistant/data_entry_flow.py

```diff
@@ -157,6 +157,7 @@ class FlowResult(TypedDict, total=False):
     result: Any
     step_id: str
     title: str
+    translation_domain: str
```

Comment:
This needs a frontend update. If the key is not present the translation_domain is the domain of the config flow, i.e. handler.